docs/docsite/rst/community/steering/community_topics_workflow.rst

```diff
@@ -4,121 +4,124 @@
    For other changes, create a `community topic <https://forum.ansible.com/new-topic?category=project&tags=community-wg>`_ to discuss them.
    (Creating a draft PR for this file and mentioning it in the community topic is also OK.)
 
-Ansible community topics workflow
-=================================
+.. _community_topics_workflow:
+
+Community topics workflow
+=========================
 
 Overview
 --------
 
-This document describes the Ansible community topics workflow (herein after ``Workflow``) to provide guidance on successful resolving topics in an asynchronous way.
+This document describes the Ansible Community Topics workflow (hereinafter Workflow) to provide guidance on successful resolving topics in the asynchronous way.
```

Comment:
Interesting, `hereinafter` really seems to be a word. I haven't seen this yet anywhere else, though. I think it's quite uncommon. Maybe simply `hereafter [just] Workflow` or `following [just] Workflow` or something similar?

Reply:
There's no reallneed here for the hereinafter clause.  As for the capitalization of Community Topics... @oraNod has been making a lot of things lowercase instead, what do you think on this one, Don?

Reply:
Removed hereinafter and capitalization, thanks